lib/src/local_backend.rs

```diff
@@ -27,6 +27,7 @@ use crate::backend::{
     ConflictId, ConflictPart, FileId, MillisSinceEpoch, Signature, SymlinkId, Timestamp, Tree,
     TreeId, TreeValue,
 };
+use crate::content_hash::ContentHash;
```

Comment:
Looks like line 76 (GitHub doesn't let you comment on unchanged lines) now needs to have "482ae5a29fbe856c7272f2071b8b0f0359ee2d89ff392b8a900643fbd0836eccd067b8bf41909e206c90d45d6e7d8b6686b93ecaee5fe1a9060d87b672101310"